cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,10 +1,11 @@
-import {MenuContentPage} from "../page/index";
-import {ProductListPage} from "../page/products-list.page";
-import {ShoppingCartPage} from "../page/shopping-cart.page";
-import {LoginPage} from "../page/login.page";
-import {AddressStepPage} from "../page/ address-step.page";
-import {ShippingStepPage} from "../page/shipping-step.page";
-import {PaymentStepPage} from "../page/ payment-step.page";
+import {MenuContentPage,
+  ProductListPage,
+  ShoppingCartPage,
+  LoginPage,
+  AddressStepPage,
+  ShippingStepPage,
+  PaymentStepPage,
```

Comment:
The PaymentStep page should contain the selection of the payment method (Bank wire) and the click on the confirm order button